Dockerfile

```diff
@@ -144,7 +159,17 @@ RUN pip install pybids==0.5.1
 ENV PYTHONPATH=""
 
 COPY run.py /run.py
-RUN chmod +x /run.py
+RUN chmod 555 /run.py
 
 COPY version /version
+COPY IntendedFor.py /IntendedFor.py
+COPY modified_files/fsl_sub /usr/lib/fsl/5.0/fsl_sub
```

Comment:
What changes are required to this file? It would be better to apply them via `diff`